### PR TITLE
feat: connects filter, cache and adds rns buy forms #8 #10

### PR DESCRIPTION
**Pull request by jurajpiar**

### CODE REVIEW

**Comment by jurajpiar** [comment]
Master-rebased.

**Comment by julianmrodri** [comment]
Noticed when running that I only see one filter for Price, the one for example for Currency are not being displayed.

File: src/components/organisms/DomainsFilters.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                            }
+                        }
+                    })
+                }}
+            />
+            {/* <SelectFilter title='Currency' items={currencyItems}/> */}
+        </>
+    )
+}
+
```
Comment (julianmrodri) [inline src/components/organisms/DomainsFilters.tsx:99]: Would be nice to at least display the Currency filter (checkboxes) so we can see it in the UI.

File: src/components/molecules/table/SelectRowButton.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react'
```
Comment (farinalvaro) [inline src/components/molecules/table/SelectRowButton.tsx:1]: 4 spaces indentation

File: src/components/organisms/DomainsFilters.tsx (hunk 1 above)
Comment (jurajpiar) [reply src/components/organisms/DomainsFilters.tsx:128]: done, but disconnected ie "dead"

**Pull request merged**